csi.proto

```diff
@@ -974,6 +987,27 @@ message ControllerGetVolumeResponse {
   // This field is REQUIRED.
   VolumeStatus status = 2;
 }
+message ModifyVolumeRequest {
+  option (alpha_message) = true;
+
+  // Contains identity information for the existing volume.
+  // This field is REQUIRED.
+  string volume_id = 1;
+
+  // Secrets required by plugin to complete modify volume request.
+  // This field is OPTIONAL. Refer to the `Secrets Requirements`
+  // section on how to use this field.
+  map<string, string> secrets = 2 [(csi_secret) = true];
+
+  // This field is OPTIONAL.This allows the CO to specify the
+  // mutable parameters to apply. 
```

Comment:
Expand this wording by copying the explanation from 386-389.

Reply:
Replace lines 1002-1008 with:

// Plugins specific parameters to apply, passed in as opaque key-value
// pairs. This field is OPTIONAL. The Plugin is responsible for 
// parsing and validating these parameters. COs will treat these 
// as opaque. COs MUST specify the intended value of every mutable
// parameter. Absent values that were previously set MUST be
// interpretted as clearing the value.

Reply:
@saad-ali @jdef this is the biggest change, can you review again?